web/templates/note/show.html.eex

```diff
@@ -0,0 +1,7 @@
+<h1>
+  <b><%= @note.title %></b> (<%= @note.id %>)
```

Comment:
This is a pattern from the book that I mimicked. It distinguishes the title from the id.